CONTRIBUTING.md

```diff
@@ -0,0 +1,96 @@
+# Contributing
+
+Contributions from the community are essential in keeping Hibernate (and any Open Source
+project really) strong and successful.  
+
+# Legal
+
+You should check the relevant license under which all contributions will be licensed for the specific project you are contributing to.
+
+All contributions are subject to the [Developer Certificate of Origin (DCO)](https://developercertificate.org/).  
+
+The DCO text is available verbatim in the [dco.txt](dco.txt) file.
+
+
+## Guidelines
+
+While we try to keep requirements for contributing to a minimum, there are a few guidelines 
+we ask that you mind.
+
+For code contributions, these guidelines include (**where applicable**):
+* Respect the project code style.
+* Have a corresponding JIRA [issue](https://hibernate.atlassian.net/browse/HHH) and be sure to include the key for this JIRA issue in your commit messages.
```

Comment:
maybe let's try to send people to the jira instance and there to find the project they need?
```suggestion
* Have a corresponding JIRA [issue](https://hibernate.atlassian.net/) and be sure to include the key for this JIRA issue in your commit messages.
```